src/unix/core.c

```diff
@@ -198,6 +201,19 @@ void uv__make_close_pending(uv_handle_t* handle) {
   handle->loop->closing_handles = handle;
 }
 
+int uv__getiovmax() {
```

Comment:
This duplicates the function of the same name in stream.c.  Can you add it to internal.h and remove it from stream.c?  While you're here, can you update the prototype to `int uv__getiovmax(void)`?  Thanks.